src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let total = 0;
+
+  for (let i = 0; i < days; i++) {
+    total += 40;
+  }
```

Comment:
Why don't just do?))
```suggestion
  let total = days * 40;
```